docs/spec/README.md

```diff
@@ -1,27 +1,31 @@
-# Cosmos Hub Spec
+# Specifications
 
-This directory contains specifications for the state transition machine of the
-Cosmos Hub. 
+This directory contains specifications for the modules of the Cosmos SDK as well as Interchain Standards (ICS) and other specifications.. 
 
-The Cosmos Hub holds all of its state in a Merkle store. Updates to
+SDK applications hold thri state in a Merkle store. Updates to
```

Comment:
thri ?

Reply:
This is a great usecase for the suggest changes github feature...